components/lib/crash/src/main/res/values/strings.xml

```diff
@@ -21,4 +21,6 @@
     <!-- Label of a notification action/button that will send the crash report to Mozilla. -->
     <string name="mozac_lib_crash_notification_action_report">Report</string>
 
+    <!-- Label of notification showing that the crash report service is running. -->
+    <string name="mozac_lib_send_crash_report_in_progress">Sending crash report to %1$s</string>
```

Comment:
👍 for adding the comment here. Those comments will show up when our translators translate the strings in Pontoon: https://pontoon.mozilla.org/projects/android-l10n/

Please mention in the comment what `%1$s` will be replaced with in the UI. This will help the translators to understand the context better. :)

Reply:
Good catch thanks.